<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlastAllSupportDialog</class>
 <widget class="QDialog" name="BlastAllSupportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>315</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Request to Local BLAST Database</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="optionsTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Select search:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="programName">
           <item>
            <property name="text">
             <string extracomment="qwe">blastn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>blastp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>gpu-blastp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>blastx</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tblastn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tblastx</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="shortSequenceCheckBox">
           <property name="text">
            <string>Search for short, nearly exact matches</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Expectation value </string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="megablastCheckBox">
           <property name="text">
            <string>Megablast</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="hitsLabel">
           <property name="text">
            <string>Max hits:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="numberOfHitsSpinBox">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="specialValueText">
            <string>Off</string>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="evalueSpinBox">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="minimum">
            <double>0.000001000000000</double>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="value">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="bothStrandsButton">
             <property name="text">
              <string>Both strands</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="directStrandButton">
             <property name="text">
              <string>Direct</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="complStrandButton">
             <property name="text">
              <string>Complement</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="dbSelectorWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Number of CPUs being used</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numberOfCPUSpinBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>6000</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="4">
          <widget class="QComboBox" name="serviceComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>plain</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>psi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>phi</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="matchScoreLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Match scores</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="serviceLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Service</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QComboBox" name="scoresComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>1 -3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 -4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 -2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1 -1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 -7</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 -5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 -3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4 -5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5 -4</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Word size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="matrixLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Matrix</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="matrixComboBox">
           <item>
            <property name="text">
             <string>BLOSUM62</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BLOSUM45</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BLOSUM80</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PAM30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PAM70</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1" colspan="4">
          <widget class="QLineEdit" name="phiPatternEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="phiPatternLabel">
           <property name="text">
            <string>PHI pattern</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="wordSizeSpinBox">
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="thresholdLabel">
           <property name="toolTip">
            <string>Threshold for extending hits</string>
           </property>
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="thresholdSpinBox"/>
         </item>
         <item row="0" column="4">
          <widget class="QComboBox" name="costsComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="costsLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Gap costs</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="compStatsLabel">
           <property name="text">
            <string>Composition-based statistics</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="4">
          <widget class="QComboBox" name="compStatsComboBox">
           <item>
            <property name="text">
             <string>D or d: default (equivalent to 2 )</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0 or F or f: No composition-based statistics</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1: Composition-based statistics as in NAR 29:2994-3005, 2001</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2 or T or t : Composition-based score adjustment as in Bioinformatics 21:902-911, 2005, conditioned on sequence properties</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3: Composition-based score adjustment as in Bioinformatics 21:902-911, 2005, unconditionally</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Filters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="lowComplexityFilterCheckBox">
              <property name="text">
               <string>Low complexity filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="repeatsCheckBox">
              <property name="text">
               <string>Human repeats filter</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Masks</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="lookupMaskCheckBox">
              <property name="text">
               <string>Mask for lookup table only</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="lowerCaseCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Mask lower case letters</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="extOptionsTab">
      <attribute name="title">
       <string>Extension options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QGroupBox" name="xDropoffGroupBox">
           <property name="title">
            <string>X dropoff value (in bits)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>For gapped alignment</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="xDropoffGASpinBox">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>For ungapped extensions</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>For final gapped alignment</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="xDropoffFGASpinBox">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="xDropoffUnGASpinBox"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Multiple Hits window size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="windowSizeSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="gappedAlignmentCheckBox">
         <property name="text">
          <string>Perform gapped alignment</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Yes</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections/>
</ui>
